script.skin.helper.colorpicker/addon.xml

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+	<addon id="script.skin.helper.colorpicker" name="Skin Helper Service ColorPicker" version="1.0.0" provider-name="marcelveldt">
+	<requires>
+		<import addon="xbmc.python" version="2.13.0"/>
+		<import addon="xbmc.addon" version="12.0.0"/>
+		<import addon="script.module.pil" version="1.1.7"/>
```

Comment:
Sorry, that file is included:
https://github.com/marcelveldt/repo-scripts/tree/script.skin.helper.colorpicker/script.skin.helper.colorpicker/resources/lib

I think it's because all these PNG files that it's too much for Github to handle in the browser